include/erlcloud_aws.hrl

```diff
@@ -20,7 +20,9 @@
 
 -record(aws_config, {
           as_host="autoscaling.amazonaws.com"::string(),
+          ec2_protocol="https"::string(),
```

Comment:
maybe it's time to generalise it for all? like we have VPCe

Reply:
Like `services_protocol` / `services_port` settings at the level of application `env` config?

Reply:
(Which would override any corresponding settings in `#aws_config{}` that were kept to their default values)

Reply:
yes. smth like: 
if env_set use env:get(*_protocol)
else if aws_config{*_protocol}
else use default

granted, probably need to handle port variances for different service so env would looks like proplist/map and not just 2 variables.

Reply:
That sounds reasonable!

Doing it for all services will bit a bit work though, and I won't find the time for it soon (the EC2 change is somewhat urgent), so I'll leave this PR open and use my fork for now.

Reply:
I did not mean to ask to rework this PR. You send a follow up. We are not heavy users of local stack, only that. 

I’ll re-review this one later and probably merge and tag.

Reply:
Ahh, alright. Thanks!